CMakeLists.txt

```diff
@@ -1199,7 +1199,7 @@ math(EXPR VERSION_NUMBER_INTEGER
 # 5.0 broke backcompat in drsyms and xmm opnd sizes
```

Comment:
Please add a line for this breakage to match these comments.

Reply:
Done.